packages/components/radio/src/use-radio.ts

```diff
@@ -117,8 +123,7 @@ export function useRadio(props: UseRadioProps) {
   } = useReactAriaRadio(
     {
       value,
-      children,
-      ...groupContext,
+      children: typeof children === "function" ? true : children,
```

Comment:
I referenced the react-aria-components implementation.

see: 
- https://github.com/adobe/react-spectrum/blob/4e3af379e569faac3b374e0ed5a98b2a19cd92c3/packages/react-aria-components/src/RadioGroup.tsx#L183-L184
- https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/radio/src/useRadio.ts#L52